README.md

```diff
@@ -183,6 +183,24 @@ The docker-compose-standalone.yml file is meant for local testing/troubleshootin
 To utilize this, pass the -f flag to the docker-compose command as follows:
 > docker-compose -f docker-compose-confluent-cloud.yml up
 
+### Data & Config Files
+Data and config files are expected to be in a location pointed to by the DOCKER_SHARED_VOLUME environment variable.
+
+At this time, the PPM assumes that this location is the /ppm_data directory. When run in a docker or k8s solution, an external drive/directory can be mounted to this directory.
+
+In a BSM configuration, the PPM requires the following files to be present in the /ppm_data directory:
```

Comment:
Should we update since fieldsToRedact is no longer required?

Reply:
Yup good call, I removed fieldsToRedact.txt from the list of required files here.